frontend/src/pages/[slug]/TrackingSlug.tsx

```diff
@@ -1,69 +1,211 @@
+import { useState } from 'react';
 import { useParams } from 'react-router-dom';
+import { PARCEL_STATUS } from '@/constants';
 
 import { useSmallScreen } from '@/hooks/useSmallScreen';
+import { useAuth } from '@/contexts/AuthContext';
 
-import { useTracking } from '@/apis/tracking';
+import { useTimeline, useTracking } from '@/apis/tracking';
 
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import EventIcon from '@mui/icons-material/Event';
+import LocalShippingIcon from '@mui/icons-material/LocalShipping';
+import UpdateIcon from '@mui/icons-material/Update';
 import Box from '@mui/material/Box';
-import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import CircularProgress from '@mui/material/CircularProgress';
+import Divider from '@mui/material/Divider';
+import LinearProgress from '@mui/material/LinearProgress';
+import Stack from '@mui/material/Stack';
+import Tooltip from '@mui/material/Tooltip';
 import Typography from '@mui/material/Typography';
 
 import { QueryStates } from '@/components/QueryStates';
 import TimelineComponent from '@/components/Timeline';
+import TrackingDetails from '@/components/TrackingDetails';
 
 function TrackingSlug() {
   const { slug } = useParams();
+  const authContext = useAuth();
+  const isLoggedIn = authContext.isAuthenticated;
```

Comment:
you can basically call and use isAuthenticated directly, without aux variable `isLoggedIn`
`  const { isAuthenticated } = useAuth();`